File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex11.c
#include <stdio.h>

main()
{
    printf("Aumento Salarial\n\n");

    //vars
    float salarioInicial, aumento, salarioFinal;

    //Entrada
    printf("Insira o salario atual do funcionario: ");
    scanf("%f",&salarioInicial);

    //Processamento
    if ( salarioInicial <= 300 )
        aumento = salarioInicial * 0.15;
    else if ( salarioInicial <= 600 )
        aumento = salarioInicial * 0.1;
    else if ( salarioInicial <= 900 )
        aumento = salarioInicial * 0.05;
    else
        aumento = 0;

    salarioFinal = salarioInicial + aumento;

    //Saída
    printf("Aumento R$ %.2f\n", aumento);
    printf("Salario final: R$%.2f\n", salarioFinal);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex11.java
//Importando Scanner
import java.util.Scanner;

public class Ex11 {

	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
		
	System.out.println("Aumento Salarial\n\n");
	
	//vars
	double salarioAtual, aumento, novoSalario;
	
	//Entrada
	System.out.print("Insira o salário atual do funcionário: R$ ");
	salarioAtual = ent.nextDouble();
	
	//Processamento
	if ( salarioAtual <= 300 )
		aumento = salarioAtual * 0.15;
	else if ( salarioAtual <= 600 )
		aumento = salarioAtual * 0.1;
	else if ( salarioAtual <= 900 )
		aumento = salarioAtual * 0.05;
	else
		aumento = 0;
	
	novoSalario = salarioAtual + aumento;
	
	//Saída
	System.out.println("Aumento: R$ " + aumento);
	System.out.printf("Novo salário: R$ %.2f", novoSalario);
	
	}
}
